.travis.yml

```diff
@@ -4,20 +4,22 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
-    - CONDA_PERL=5.20.3.1
+    - CONDA_PERL=5.20.3.1  CONDA_PY=27
+    - CONDA_PERL=5.22.0.1  CONDA_PY=27
+    - CONDA_PERL=5.22.2.1  CONDA_PY=27
```

Comment:
Obviously we don't need to build against every version of `perl`. May be worth having a discussion out of band on this.

xref: https://github.com/conda-forge/conda-smithy/issues/415